### DISCUSSION TITLE
Enabling move_script_to_bottom is breaking the layout

### DISCUSSION THREAD

**Discussion by patruvlad** [breezefront/community#339] [1. Help] [answered]
Hello we are close to go to production and we wanted to test in a Production mode how things work but when enabling move_script_to_bottom the layout breaks and with a bit of inspecting it looks like when this is enabled elements (eveerything except page-header) get moved outside the page-wrapper container.
Magento version 2.4.7-p1 
![image](https://github.com/user-attachments/assets/35485e1a-f2a8-40e8-9b47-2f489b89b30a)

**Comment by vovayatsyuk** [comment]
I don't know why it's broken. Try disabling `move_script_to_bottom` and verify if your markup is valid.

**Reply by patruvlad** [reply]
Disabling `move_script_to_bottom` solves the markup issue.

**Reply by vovayatsyuk** [reply]
Please provide a URL to the page that gets broken when you enable this option.

**Reply by patruvlad** [reply]
hello, sorry for the late reply here is the link to the store with the issue https://m2-demo.sumashop.ro/

**Reply by vovayatsyuk** [reply]
Thanks! Can you please disable move to the bottom now?

**Reply by patruvlad** [reply]
sure, done. I saw that this happens in developer mode also , not only production mode

**Reply by vovayatsyuk** [reply, marked as answer]
You have scripts inside svg tag of your custom icons. Maybe this is the reason?

![CleanShot 2024-11-15 at 14 20 03@2x](https://github.com/user-attachments/assets/f8d06dd9-2a91-4127-b0eb-55e24a1fb116)

Try to remove them and see if it helps.

Meanwhile, I'll try to reproduce this issue locally and see if I can do something to keep the site working with these scripts inside svg.

**Reply by patruvlad** [reply]
That was the problem, looks like the SVG provider for some reason adds <script> tag inside it. thanks for the support

**Reply by vovayatsyuk** [reply]
Can't reproduce this issue. Works well on my local setup. Maybe, I have a newer version of libxml library and it's more stable. I don't know..
